### PR TITLE
docs: address QS report docs

### PR DESCRIPTION
**Pull request by prestwich**
Address doc issues from QS report

## PR Checklist

- [n/a] Added Tests
- [x] Updated Documentation
- [n/a (non-functional)] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/contracts-core/contracts/Replica.sol [hunk 1]
```diff
@@ -109,6 +109,19 @@ contract Replica is Version0, NomadBase {
 
     // ============ Initializer ============
 
+    /**
+     * @notice Iniitalize the replica
```
Comment (odyslam) [inline packages/contracts-core/contracts/Replica.sol:113]:
```suggestion
     * @notice Initialize the replica
```

File: packages/contracts-bridge/contracts/TokenRegistry.sol [hunk 2]
```diff
@@ -341,7 +341,9 @@ contract TokenRegistry is Initializable, XAppConnectionClient, ITokenRegistry {
     {
         // deploy and initialize the token contract
         _token = address(new UpgradeBeaconProxy(tokenBeacon, ""));
-        // initialize the token separately from the
+        // Initialize the token. Normally we initialize during proxy deployment
+        // but due to a proxy quirk, it's clearer to do it explicitly when
```
Comment (odyslam) [inline packages/contracts-bridge/contracts/TokenRegistry.sol:345]: what's a "proxy quirk"? Perhaps we can add a sentence or point the user somewhere

File: packages/contracts-core/contracts/Home.sol [hunk 3]
```diff
@@ -51,11 +51,13 @@ contract Home is Version0, QueueManager, MerkleTreeManager, NomadBase {
...
      * @notice Emitted when a new message is dispatched via Nomad
+     * @param messageHash Hash of message; the leaf inserted to the Merkle tree
+     *        for the message
      * @param leafIndex Index of message's leaf in merkle tree
      * @param destinationAndNonce Destination and destination-specific
-     * nonce combined in single field ((destination << 32) & nonce)
-     * @param messageHash Hash of message; the leaf inserted to the Merkle tree for the message
-     * @param committedRoot the latest notarized root submitted in the last signed Update
+     *        nonce combined in single field ((destination << 32) & nonce)
+     * @param committedRoot the latest notarized root submitted in the last
```
[the remark this answers is not in the document]
Comment (prestwich) [reply packages/contracts-core/contracts/Home.sol:59]: oh I prefer hanging indents to match the first meaningful word of the previous line. i.e. make it visually obvious it's a continuation line

File: packages/contracts-bridge/contracts/TokenRegistry.sol (hunk 2 above)
Comment (prestwich) [reply packages/contracts-bridge/contracts/TokenRegistry.sol:345]: the proxy quirk is that if the initdata is length 0, initialize is not called

Comment (prestwich) [reply packages/contracts-bridge/contracts/TokenRegistry.sol:345]: don't think we need to specify more

Comment (anna-carroll) [reply packages/contracts-bridge/contracts/TokenRegistry.sol:345]: for the record i think it makes more sense for a proxy to not call initialize if there's no initialize calldata passed in. what if the implementation doesn't *have* an initialize function?

Comment (anna-carroll) [reply packages/contracts-bridge/contracts/TokenRegistry.sol:345]: so this should really saw "due to awesome proxy design"

**Review by anna-carroll — APPROVED** [review]

**Pull request merged**